lnplus/exports.sh

```diff
@@ -0,0 +1,3 @@
+export APP_LNPLUS_IP="10.21.21.77"
+export APP_LNPLUS_PORT="3777"
+export APP_LNPLUS_RAILS_ENVIRONMENT="production"
```

Comment:
If you make the changes within the `docker-compose.yml` file, then this `exports.sh` file is no longer neccessary and simplifies your app :)